packages/just-scripts/src/tasks/__tests__/tscTask.spec.ts

```diff
@@ -1,14 +1,22 @@
 import * as mockfs from 'mock-fs';
-import { encodeArgs, exec, spawn } from 'just-scripts-utils';
+import { encodeArgs, exec, spawn } from '../../utils';
 import { TaskFunction } from 'just-task';
 import { tscTask, tscWatchTask, TscTaskOptions } from '../tscTask';
 import { callTaskForTest } from './callTaskForTest';
 import { normalizeCmdArgsForTest } from './normalizeCmdArgsForTest';
 
 // Jest will hoist these before the imports above, so these modules will be mocked first
-jest.mock('just-scripts-utils/lib/exec', () => {
-  const { mockExecFactory } = require('./mockExecFactory');
-  return mockExecFactory();
+jest.mock('../../utils', () => {
+  const originalModule = jest.requireActual('../../utils');
```

Comment:
This was moved from the `mockExecFactory` file (which was very simple and only used in one place)